### PR TITLE
Feature/feature fix graphics

### PR DESCRIPTION
**Pull request by xavier-campos-diaz**

### CODE REVIEW

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/features/pets/PetHealthInfo.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5541824&bid=18169023)

File: app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/infopet/InfoPetHealthFragment.java [hunk 1]
```diff
@@ -35,7 +35,7 @@
     private FragmentInfoPetHealthBinding binding;
     private TextView statisticTitle;
     private MaterialButton btnAddNewStatistic;
-    private BarChart barChart;
+    private static BarChart barChart;
```
Comment (santidrj) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/infopet/InfoPetHealthFragment.java:38]: Codacy Issue found: [Static variable definition in wrong order.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5541824)

**Review by AlbertPG — APPROVED** [review]
Okey, I see no problems

**Review by enrichp — APPROVED** [review]

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ app/src/main/java/org/pesmypetcare/mypetcare/features/pets/PetHealthInfo.java  2
         

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/features/pets/PetHealthInfo.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5541824&bid=18169023)

**Pull request merged**